api/scripts/certification/rescore-certifications.js

```diff
@@ -0,0 +1,43 @@
+import 'dotenv/config';
+
+import * as url from 'node:url';
+
+import { disconnect, knex } from '../../db/knex-database-connection.js';
+import { CertificationRescoringByScriptJob } from '../../lib/infrastructure/jobs/certification/CertificationRescoringByScriptJob.js';
+import { logger } from '../../src/shared/infrastructure/utils/logger.js';
+
+const modulePath = url.fileURLToPath(import.meta.url);
+const isLaunchedFromCommandLine = process.argv[1] === modulePath;
+
+async function main(certificationCourseIds) {
+  await _scheduleRescoringJobs(certificationCourseIds);
+}
+
+(async () => {
+  if (isLaunchedFromCommandLine) {
+    try {
+      const certificationCourseIds = process.argv[2].split(',');
```

Comment:
Comme on est quand même sur un truc invoqué à la main depuis une console avec un aegument probablement copié-collé de quelque part d'autre, je propose de nettoyer un peu notre process argv

```suggestion
      const certificationCourseIds = process.argv[2].split(',').map(Number.parseInt).filter(Number.isInteger);
```